build.gradle

```diff
@@ -26,6 +26,9 @@ dependencies {
 	runtimeOnly 'com.h2database:h2'
 	annotationProcessor 'org.projectlombok:lombok'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	
+	implementation group: 'org.apache.logging.log4j', name: 'log4j-api', version: '2.17.1'
+    implementation group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.17.1'
```

Comment:
Fix spacing. Also you can use the shorter notation with gradle: `"org.apache.logging.log4j:log4j-core:2.17.1"`.
You could also make 2.17.1 a variable in gradle.properties.